### PR TITLE
[luci] Support INT4/UINT4

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise to enable INT4/UINT4 data type.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for  #12717 , from draft #12743

**Review by shs-park — APPROVED** [review]
LGTM!
=)

File: compiler/luci/import/src/Nodes/CircleConst.cpp [hunk 1]
```diff
@@ -67,6 +67,54 @@ void copy_data(const VectorWrapper<uint8_t> &raw_data, uint32_t num_elements,
...
+  uint32_t raw_size = (num_elements + 1) / 2;
+  assert(raw_data.size() == raw_size);
+
+  const uint8_t *data = raw_data.data();
+  const_node->size<loco::DataType::U4>(num_elements);
+  for (uint32_t i = 0; i < raw_size; ++i)
+  {
+    uint32_t idx = i * 2;
+    // 1bit for sign, 3bit for value
+    const_node->at<loco::DataType::U4>(idx++) = static_cast<uint8_t>(data[i] << 4) >> 4;
```
Comment (ljwoo94) [inline compiler/luci/import/src/Nodes/CircleConst.cpp:112]: Sorry for comment on merged PR.
I'm just curious about `uint8`. If this data is considered as unsigned, I think we do not have to shift twice but just use 
`data[i] & 0xF`

Comment (seanshpark) [reply compiler/luci/import/src/Nodes/CircleConst.cpp:112]: #12803 is what I wanted to do someday...

**Pull request merged**